packages/manager/modules/adp/src/adp.scss

```diff
@@ -0,0 +1,52 @@
+// override bootstrap map to add custom widths
+// This variable affects the `.h-*` and `.w-*` classes.
+$sizes: () !default;
+// stylelint-disable-next-line scss/dollar-variable-default
+$sizes: map-merge(
+  (
+    10: 10%,
+    15: 15%,
+    20: 20%
+  ),
+  $sizes
+);
+
+@import "~bootstrap4/scss/_functions";
+@import "~bootstrap4/scss/_variables";
+@import "~bootstrap4/scss/_mixins";
+@import '~bootstrap4/scss/_utilities.scss';
+@import "~bootstrap4/scss/_grid.scss";
+@import "~bootstrap4/scss/_type.scss";
+@import "~bootstrap4/scss/_modal.scss";
+
+.storage-list {
```

Comment:
Should be scoped and in `./src/deploy` (the only file using it is [`src/deploy/storage.html`](https://github.com/ovh-ux/manager/pull/335/commits/235571d404a65d3856ae865f5aadad0877ad64e9#diff-0591476736ab4c76cb6efb532661c7c2R32))

Reply:
The bootstrap css imports are currently un-scoped, since it affects the modal dialog boxes too. the rest of the code has been moved to deploy.scss and scoped